File: frontend/src/utils/urls.js
export const USERS_GET_URL = "http://127.0.0.1:8000/app/get-users/";
export const PRODUCTS_GET_URL = "http://127.0.0.1:8000/app/get-products/";
export const GET_CART_ITEM_URL = "http://127.0.0.1:8000/app/get-cart-items/";

export const addItemCartUrl = pid =>
	`http://127.0.0.1:8000/app/add-item-cart/${pid}/`;

export const deleteItemCartUrl = pid =>
	`http://127.0.0.1:8000/app/delete-item-cart/${pid}/`;

File: backend/app/urls.py
from django.urls import path
from .views import *

urlpatterns = [
    path("get-users/", get_users),
    path("get-products/", get_products),
    path("add-item-cart/<int:product_id>/", add_cart_item),
    path("delete-item-cart/<int:product_id>/", delete_cart_item),
    path("get-cart-items/", get_cart_items),
]
